labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Open Enterprise Agent
+parent: Labs
+---
+# Open Enterprise Agent
+# Short Description
+The Open Enterprise Agent is a software agent for building decentralised identity applications and services in line with W3C standards and Hyperledger Aries RFCs. Aiming to be blockchain agnostic and interoperable across two key technology ecosystems in SSI, it provides a scalable, robust, performant and user-friendly platform for building solutions and products based on it. Its REST API enables easy integration with any programming language, on top of a core built in Scala, a language well suited for building fault-tolerant, highly concurrent systems.
```

Comment:
Wanted to confirm does SSI here stands for "Self-sovereign identity". 
It would be better if you can specify it in the Description along with abbreviation.

Reply:
Updated this.